packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -18,40 +18,52 @@ export enum AlertVariant {
   default = 'default'
 }
 
+/** Acts as the main component for an alert. Can also act as a wrapper around text
```

Comment:
It might be more concise to say something like "The primary/main alert component"

I'm also not sure I understand what you're describing when you say it acts as a wrapper to provide a description. Maybe you need to expound a little more on the context you're describing here?

Reply:
Good call. It was essentially just duplicating the `children` prop description, will update to just "The main alert component."